x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/alert_summary_widget_ui.tsx

```diff
@@ -5,19 +5,45 @@
  * 2.0.
  */
 
+import { ALERT_STATUS_ACTIVE, ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils';
 import { euiLightVars } from '@kbn/ui-theme';
-import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
-import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React, { MouseEvent } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { AlertsSummaryWidgetUIProps } from './types';
 
+const onClickWrapper = (
```

Comment:
I would keep this handler inside the component.

```suggestion
export const AlertsSummaryWidgetUI = ({
  active,
  recovered,
  timeRange,
  onClick,
}: AlertsSummaryWidgetUIProps) => {

  const handleClick = (event: MouseEvent<HTMLAnchorElement | HTMLDivElement>) => {
    event.preventDefault();
    event.stopPropagation();
    onClick();
  }
  
  return (
    <EuiPanel
      ...
      onClick={handleClick}
    >
     ...
  )
}
```

Reply:
Done